pom.xml

```diff
@@ -17,6 +17,8 @@
     <packaging>pom</packaging>
 
     <properties>
+        <!--suppress XmlUnresolvedReference --> <!-- git.commit.time is generated by the git-commit-id-maven-plugin-->
+        <project.build.outputTimestamp>${git.commit.time}</project.build.outputTimestamp>
```

Comment:
Does this affect the BOM? The BOM doesn't use this parent, so it won't have a project.build.outputTimestamp?  Is that important?

Reply:
Should probably add it to the boom but given it's not a jar I don't think it's actually affected

Reply:
I think you are right, it is seems unimportant.